content/cloud/changelog/spin-cloud-v0.4.0.md

```diff
@@ -0,0 +1,31 @@
+title = "Fermyon Serverless AI (Private Beta)"
```

Comment:
THis title seems to be cloned from the prior changelog entry, should be updated to be about spin cloud plugin